### ISSUE TITLE
Interface with geo

### ISSUE THREAD

**Issue by Farkal** [wyyerd/vectortile-rs#2]
Would be great to be able to use geo type with this lib

**Comment by kestred** [comment]
@Farkal - Definitely that would be great!

The current implementation relies on https://github.com/andelf/rust-postgis. There is an idea there to support geo's types, but IIRC it depends on development that hasn't been designed yet.
If that was worked through, it'd be very easy to support geo types here as well.
(See maybe https://github.com/georust/geo/issues/67 to help push that forward)

Alternatively, we could implement our own traits in `vectortile` that abstracts of `postgis` and `geo` crate types.  Generally, I'd prefer if we could implement those traits in `geo` rather than `vectortile` however, as they'll be more widely applicable.

**Comment by Farkal** [comment]
Yeah would be great to just use geotype for all lib interacting with geo products.
I just ask about the state of the issue you give me.
